backend/prisma/schema.prisma

```diff
@@ -59,20 +65,25 @@ model Question {
 }
 
 model Book {
-  id     Int @id @default(autoincrement())
-  number Int
-
-  pages   Page[]
-  courses BookCourse[]
+  id          Int           @id @default(autoincrement())
+  bookCover   String?
+  title       String
+  blurb       String?
+  author      String
+  gradeRange  String
+  category    BookCategory  // or String if you prefer
+  pages       Page[]
+  courses     BookCourse[]
 }
 
 model Page {
-  id     Int @id @default(autoincrement())
-  number Int
-
-  book   Book @relation(fields: [bookId], references: [id])
-  bookId Int
-  questions Question[]
+  id          Int           @id @default(autoincrement())
+  content     Json          // or String[]
```

Comment:
Since id is auto increment we need a way to remember the order within each book either with a new column for nextPageId  or just pageNum.